ee/tuf/autoupdate.go

```diff
@@ -233,6 +268,53 @@ func (ta *TufAutoupdater) Interrupt(_ error) {
 	ta.interrupt <- struct{}{}
 }
 
+func (ta *TufAutoupdater) Do(data io.Reader) error {
```

Comment:
maybe leave a comment over Do that it's satisfying the actor interface from ActionQueue or make a separate struct for "update now"